opsdroid/events.py

```diff
@@ -413,6 +417,37 @@ def __init__(self, name=None, params=None, *args, **kwargs):  # noqa: D107
         self.room_params = params or {}
 
 
+class Video(File):
+    """Event class specifically for video files."""
+
+    async def save_to_disk(self, fbytes):
+        """ Save the bytes to disk as video."""
+
+        temp_dir = tempfile.gettempdir()
+        file_name = "opsdroid_vid_" + await str(random.randint(0, 100000000))
+        file_path = os.path.join(temp_dir, file_name)
+
+        # Checks and deletes the output file
+        # You cant have a existing file or it will through an error
+        if os.path.isfile(file_path):
+            os.remove(file_path)
+
+        # opens the file which is accessable as 'out_file'
+        async with open(file_path, "wb") as out_file:  # open for [w]riting as [b]inary
+            await out_file.write(fbytes)
```

Comment:
Have a look at [`NamedTemporaryFile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) it exists for exactly this use case.